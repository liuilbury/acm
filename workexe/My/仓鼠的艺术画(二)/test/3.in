30 10
19 25 10
2 29 9
19 30 9
9 25 2
9 24 5
6 14 6
27 28 9
13 28 5
9 22 8
12 20 6
