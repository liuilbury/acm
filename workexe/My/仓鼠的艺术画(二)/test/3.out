0 18 18 18 18 30 30 30 48 48 48 60 60 60 60 52 52 52 52 52 40 40 24 28 28 28 28 64 36 18 